lib/ts/recipe/recipeModule/types.ts

```diff
@@ -12,20 +12,28 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-import { NormalisedAppInfo, PostAPIHookFunction, RecipePreAPIHookContext } from "../../types";
+import { NormalisedAppInfo, RecipePostAPIHookContext, RecipePreAPIHookContext } from "../../types";
```

Comment:
`RecipePostAPIHookContext, RecipePreAPIHookContext` can be defined inside this file as it's specific to a recipe

Reply:
Done